host/class/cdc/esp_modem_usb_dte/idf_component.yml

```diff
@@ -1,5 +1,5 @@
 ## IDF Component Manager Manifest File
-version: "1.2.0"
```

Comment:
Suggest changing to `version: "1.2.1"`, as it's a bugfix

Reply:
updated to `v1.2.1`